Index: src/main/resources/aplication.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/aplication.yml b/src/main/resources/aplication.yml
deleted file mode 100644
--- a/src/main/resources/aplication.yml	
+++ /dev/null	
@@ -1,8 +0,0 @@
-spring:
-  security:
-    oauth2:
-      client:
-        registration:
-          google:
-            client-id: 1060128516661-npqls8ltd5gcc8f3hv837p7il5n96dlr.apps.googleusercontent.com
-            client-secret: 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
\ No newline at end of file
Index: src/main/resources/templates/accessDenied.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/templates/accessDenied.html b/src/main/resources/templates/accessDenied.html
--- a/src/main/resources/templates/accessDenied.html	
+++ b/src/main/resources/templates/accessDenied.html	
@@ -21,7 +21,7 @@
         <h1>No tienes acceso a esta zona</h1>
 
         <h2>Prohibido ingresar</h2>
-
+        <a href="javascript: history.go(-1)" class="btn btn-success" role="button">Volver</a>
     </div>
 
 </main>
